projects/cm_mcu/ZynqMonTask.c

```diff
@@ -528,9 +548,9 @@ void zm_send_data(struct zynqmon_data_t data[])
 {
   // https://docs.google.com/spreadsheets/d/1E-JD7sRUnkbXNqfgCUgriTZWfCXark6IN9ir_9b362M/edit#gid=0
   for (int i = 0; i < ZMON_VALID_ENTRIES; ++i) {
```

Comment:
this link is obsolete I think

Reply:
Removed. 
Done.

Reply:
Though, do you think that we should make a v2 spreadsheet after this PR is done?

Reply:
we should consider having some automated way of doing this, though I think it is not trivial